tests/publishTests.js

```diff
@@ -431,7 +431,9 @@ console.log("\n*** publish.clean ***");
 
 var cleanFixture = [{
     moduleRoot: "./",
-    cleanCmd: "clean"
+    checkFileTracking: "checkFileTracking",
+    cleanCmd: "clean",
+    filesToClean: "package.json, package-lock.json"
 }];
```

Comment:
@the-t-in-rtf I have added just the options for the changes I have made. On running `npm test`, I am getting an assertion error in publish.clean